README.md

```diff
@@ -1,27 +1,43 @@
-# cidr-filtering-cni
+# bridge-filtering
 
-This plugin allows user to define allow list for ingress and egress packet filtering for a Kubernetes pod.
+This CNI plugin allows the user to control traffic flow at the IP address or port level (OSI layer 3 or 4) for particular applications in the Kubernetes cluster, thus specifying how a pod is allowed to communicate with various network "entities". The pod interface must be a port of a bridge (for example a kubevirt virtual machine connected with bridge binding method). By default, all ingress and egress traffic is denied.
+Users may create special `ConfigMaps` in the pod namespace to indicate allowed ingress or egress connections.
+This can be done for network layers L3 and L4. The supported L4 protocols are UDP and TCP.
+
+Since the nftable rules implementing traffic filtering are created when a pod is being created, the CNI cannot update the provisioned rules if those are updated in the `ConfigMaps`.
+If a configuration in a `ConfigMap` is changed, all the pods that use it need to be re-created, in order to have up-to-date configuration.
+
+> **_NOTE:_**  The CNI plugin doesn't currently support scenarios that involve IPAM assigning an IP on the pod interface. 
 
 ## Requirements
 
 - jq
 - nftables
 - sha1sum
 
-## Usage
+## Installation
+
+To install the CNI plugin, create the `manifests/daemonset.yaml` resource.
+
 
-When using the cidr-filtering-cni plugin, all ingress and egress traffic is dropped by default.
-To allow specific ingress and egress CIDR blocks/ports, create configMaps(s) referencing a NetworkAttachmentDefinition by having a label the NetworkAttachmentDefinition spec name.
-Each configMap additionally needs to have "cidr-filtering-cni" label.
+## Limitations
 
-See the example below:
+The plugin is currently not using conntrack to allow response traffic. The user is expected to configure ingress/egress policies accordingly.
+That means that for clients, egress configuration may filter desired IP subnets and ports and ingress configuration should allow all traffic or filter by IP subnet
+without filtering by ports.
+Vice versa for servers.
```

Comment:
I would simplify this sentence - I personally get a bit confused when reading it. I think it has too many words.

How about something along the lines of:
```
Policies applying to clients (in a client/server architecture) must remember to accept the return traffic for their requests.
```
I wouldn't even mention the vice-versa for servers part.

Reply:
[Thanks, Done](https://github.com/rhrazdil/cidr-filtering-cni/pull/13/commits/9fb106b692fb73da4518627a34020acbfc01a52b)